apps/aehttp/src/aehttp_app.erl

```diff
@@ -11,7 +11,7 @@
 -define(DEFAULT_SWAGGER_EXTERNAL_PORT, 8043).
 -define(DEFAULT_SWAGGER_INTERNAL_PORT, 8143).
 -define(DEFAULT_WEBSOCKET_INTERNAL_PORT, 8144).
--define(INT_ACCEPTORS_POOLSIZE, 100).
+-define(INT_ACCEPTORS_POOLSIZE, 10).
```

Comment:
I see that this change gets lost after regenerating Swagger, though this seems inevitable at the moment as it is [hardcoded in upstream codegen](https://github.com/swagger-api/swagger-codegen/blob/28467a5c3f86eac33aa5391a024f4fbf62364646/modules/swagger-codegen/src/main/resources/erlang-server/server.mustache#L4). Ok.